cypress/integration/createNewUser.js

```diff
@@ -0,0 +1,47 @@
+/// <reference types="cypress" />
+import { faker } from "@faker-js/faker";
+
+describe("Testing for add new customer ", () => {
```

Comment:
You can write the describe test block description as :Test for adding new customer